server/src/main/scala/org/http4s/server/middleware/CORS.scala

```diff
@@ -195,7 +195,7 @@ object CORS {
     "Depends on a deficient `CORSConfig`. See https://github.com/http4s/http4s/security/advisories/GHSA-52cf-226f-rhr6. If config.anyOrigin is true and config.allowCredentials is true, then the `Access-Control-Allow-Credentials` header will be suppressed starting with 0.22.3.",
     "0.21.27",
   )
-  @nowarn("cat=deprecation")
+  @nowarn212("cat=deprecation")
```

Comment:
```scala
@nowarn("cat=deprecation")
```
Without this, the build fails in Scala 2.12. With this, the build fails in all the other versions.

So I had to use
```scala
@nowarn212("cat=deprecation")
```
from `scalac-compat-annotation`.